cmd/edgectl/aes_install.go

```diff
@@ -735,8 +752,13 @@ func (i *Installer) Perform(kcontext string) error {
 		return err
 	}
 
+	// Add a little color
+
 	i.show.Println()
-	i.ShowWrapped(fmt.Sprintf(fullSuccess, i.hostname))
+	i.ShowWrapped(color.Info.Sprintf(fullSuccess,
+		chalk.Bold.TextStyle(i.hostname),
+		chalk.Bold.TextStyle("edgectl install "+i.hostname)))
+
```

Comment:
I think this should be `login` not `install`.

Reply:
Oh, and the namespace flag... You have it correct in the comment below.

Reply:
I wasn't sure since reading elsewhere in the code says that `--namespace ambassador` is the default.  Do we want to always have our users specify the namespace?

Reply:
Ah yes, for `edgectl login` the default for the namespace argument works fine. You are right.

Reply:
resolved, fixed `login` and removed the namespace text.